src/status_im2/contexts/wallet/address_watch/view.cljs

```diff
@@ -1,6 +1,7 @@
 (ns status-im2.contexts.wallet.address-watch.view
```

Comment:
perhaps the ns `status-im2.contexts.wallet.select-address-to-watch.view` is clearer here and then
 the following page can be 
 `status-im2.contexts.wallet.add-watch-only-account.view`